### PR TITLE
Added luis error logging. Resolves #181.

### LINKED ISSUE #181: Show LUIS response error messages when exception in LUIS Client occurs.
Show LUIS response error messages when exception in LUIS Client occurs. It is hard to debug right now when using in Azure DevOps pipeline tasks.

### PR DESCRIPTION
**Pull request by in4margaret**

### CODE REVIEW

File: src/NLU.DevOps.LuisV3/LuisNLUTestClient.cs [hunk 1]
```diff
@@ -40,15 +46,23 @@ public LuisNLUTestClient(LuisSettings luisSettings, ILuisTestClient luisClient)
...
-            var luisResult = await this.LuisClient.QueryAsync(predictionRequest, cancellationToken).ConfigureAwait(false);
-            return this.LuisResultToLabeledUtterance(new SpeechPredictionResponse(luisResult, 0));
+                var predictionRequest = query.ToObject<PredictionRequest>();
+                predictionRequest.Query = predictionRequest.Query ?? query.Value<string>("text");
+                var luisResult = await this.LuisClient.QueryAsync(predictionRequest, cancellationToken).ConfigureAwait(false);
+                return this.LuisResultToLabeledUtterance(new SpeechPredictionResponse(luisResult, 0));
+            }
+            catch (ErrorException ex)
+            {
+                Logger.LogError(ex.Body.ErrorProperty.Message);
```
Comment (rozele) [inline src/NLU.DevOps.LuisV3/LuisNLUTestClient.cs]: Can you also log the HTTP status code and any other pertinent error details, e.g.:
```
Logger.LogError($"Received error with status code '{ex.Body.StatusCode}' and message '{ex.Body.Message}'.");
```

Comment (in4margaret) [reply src/NLU.DevOps.LuisV3/LuisNLUTestClient.cs]: done.

**Comment by in4margaret** [comment]
#238

**Comment by rozele** [comment]
Looks like this needs rebase

**Comment by rozele** [comment]
Still has merge conflicts.

**Pull request merged**